app/models/billboard.rb

```diff
@@ -0,0 +1,23 @@
+class Billboard < ApplicationRecord
+  has_many :requests
+	belongs_to :user
+	has_one_attached :image
+  before_create :image_nil 
+  geocoded_by :address 
+  after_validation :geocode, if: :address_changed?
+
+	def image_nil
```

Comment:
That's a bad name for the method. Should be something like `attach_empty_image` or `attach_dummy_image`